src/main/java/seedu/guest/model/GuestPrefs.java

```diff
@@ -11,28 +11,28 @@
 /**
```

Comment:
Should we keep this file as UserPrefs instead of GuestPrefs? User should be the hotel manager using the app, not the guests.